adobe/spectrum-web-components
# build: use lerna 8 and still publish react wrappers

## Description
- update to latest lerna
- rework the publish script to prevent the fact that `/react` is `.gitignored` by default from preventing lerna from publishing it

## How has this been tested?
    1. Do a publish
    2. Make sure it works
    3. Make sure that `/react` isn't _actually_ added to the project

## Types of changes

## Checklist
